src/main/resources/fabric.mod.json

```diff
@@ -23,7 +23,7 @@
     "mixins.reach-entity-attributes.json"
   ],
   "depends": {
-    "minecraft": ">=1.19",
+    "minecraft": "~1.19.3",
```

Comment:
How does `~` function in Fabric's versioning schema?

Reply:
Anything above and including 1.19.3, and below and excluding 1.20